source/standards/source-code/use-github.html.md.erb

```diff
@@ -1,63 +1,69 @@
 ---
 title: Use GitHub
-last_reviewed_on: 2024-11-20
+last_reviewed_on: 2025-07-19
 review_in: 6 months
 ---
 
 # <%= current_page.data.title %>
 
-GDS uses the [alphagov] organisation on [GitHub] to collaborate on code. The GOV.UK One Login programme uses [govuk-one-login].
+GDS uses [GitHub] to collaborate on code.
+We have two GitHub organisations, the GDS Product group use [gds-alphagov].
+The GOV.UK One Login programme uses [gds-govuk-one-login].
 
-### Getting access to `alphagov`
+## Common rules of access
 
-You can use your personal GitHub account to access alphagov if you wish.
+You can use your personal GitHub account to access either of the GDS GitHub organisations.
 
-All github user accounts added to `alphagov` *must* be [connected](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account) with a valid `digital.cabinet-office.gov.uk` email address. Accounts not connected to a valid email address will be removed from `alphagov`.
+All GitHub user accounts *must* be [connected][github-work-emails-for-github] with a valid `digital.cabinet-office.gov.uk` email address.
+Accounts not connected to a valid email address will be removed from the organisation.
 
 You must also set up two-factor authentication on your account.
 
-To join `alphagov` ask your tech lead or technical architect to invite you. Make sure you've connected your GDS email address to your account first, otherwise your account will be removed.
+### Getting access to `alphagov`
```

Comment:
Maybe these sections need to mention again the common req's in case people skip to them

Reply:
Humm, sorry this is old and I think I'd like to move ahead right now, I trust folks to read up - if we're getting inappropriate alphagov requests, happy to modify